blkn/src/SystemMessage/IconWrapper.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+import * as React from "react";
+import { colors } from "../constants";
+
+type Props = {
+  type: "warning" | "error" | "success" | "info",
+  icon: Function,
+};
+
+const fillColors = {
+  warning: "#f9971e",
+  error: colors.red,
+  success: colors.green,
+  info: colors.azure,
+};
+
+const IconWrapper = (props: Props) => {
+  const { icon: Icon, type } = props;
```

Comment:
`Icon` used as a class, make the prop PascalCase

Reply:
I really do not understand this comment :(, what should be PascalCased?

Reply:
Ok I read your comment from other PR now it's clear thank you

Reply:
After long fight with this problem: this just does not work when props is PascalCased. I cannot do antyhing about it.

Reply:
how doesn't it work? I've never had problems with pascal cased props